rules/ke_scanner_rules.yaml

```diff
@@ -106,3 +56,181 @@ rules:
 #     mode: whitelist
 #     values:
 #       - ["COS", "COS"]
+
+# Proof of concept CIS GCP benchmark rules below.
```

Comment:
Can we add this CIS disclaimer then we can submit. 

 # This scanner has not been validated by the formal CIS certification process.